docs/setup.md

```diff
@@ -11,82 +11,95 @@
 
 ## Setup Outbreak Response Framework on FortiSOAR
-After installation of **Outbreak Response Framework** solution pack, run the configuration wizard to ready your FortiSOAR environment to investigate Outbreak Alerts. This wizard helps you select the configured "Threat Detection Integrations" and "Ticketing/ITSM Integrations" on FortiSOAR.
+After installation of the **Outbreak Response Framework** solution pack, run the configuration wizard to ready your FortiSOAR environment to investigate Outbreak Alerts. This wizard helps you select the configured "Threat Detection Integrations" and "Ticketing/ITSM Integrations" on FortiSOAR.
 
-1. Log in to FortiSOAR and [after installation](#installation), click the button **Configure** from the lower-left of the screen.
+1. Log in to FortiSOAR, after [installation](#installation) completes.
+2. Click the button **Configure** from the lower-left of the screen.
 
     ![Outbreak Response start configuration](./res/config-wizard-00.png)
 
-2. Click the button **Let's get started** on the Outbreak Response Framework configuration page.
+3. Click the button **Let's get started** on the Outbreak Response Framework configuration page.
 
     ![Outbreak Alert get started](./res/config-wizard-01.png)
 
-3. Specify Threat Detection Integrations sources to run outbreak response hunt activities. Click **Next** to proceed.
+4. Select Threat Detection Integration sources to run outbreak response hunt activities and click **Next**.
 
     ![Define your Threat Detection Integrations](./res/config-wizard-02.png)
 
-4. Specify Ticketing/ITSM Integrations sources as part of your response or threat management strategy. Click **Next** to proceed.
+5. Select Ticketing/ITSM Integration sources as part of your response or threat management strategy and click **Next**.
 
     ![Define your Ticketing/ITSM Integrations](./res/config-wizard-03.png)
 
-5. Click **Finish** to complete the configuration process.
+6. Click **Finish** to complete the configuration process.
     ![All set](./res/config-wizard-04.png)
 
-## Configure the Threat Detection Integrations Parameters
+## Configure the Threat Detection Integration Parameters
 
-Following are some prerequisites for using this solution pack:
-Open "Investigate Outbreak" playbook from "10 - SP - Outbreak Response Framework" playbook collection and add following configuration parameters in set variable step for the "Threat Detection Integrations" that you have selected in configuration wizard.
+1. The following are some prerequisites for using this solution pack:
 
-- Fortinet FortiSIEM Config: 
-![Fortinet FortiSIEM Config](./res/fsm_config.png)
+2. Open the **Investigate Outbreak** playbook from the *10 - SP - Outbreak Response Framework* playbook collection.
+Add the following configuration parameters in the *Set Variable* step depending on the *Threat Detection Integrations* that you have selected in the **Configuration** wizard.
```

Comment:
It will be good if we also explain them here. Any thoughts why these are not described here?